src/main/java/me/totalfreedom/totalfreedommod/command/Command_unbanname.java

```diff
@@ -48,9 +48,12 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
 
         if (!silent)
         {
-            FUtil.staffAction(sender.getName(), "Unbanned the name " + name, true);
+            FUtil.staffAction(sender.getName(), "Removed the name " + name + " from the ban list.", true);
+        }
+        else
+        {
+            msg("Quietly removed the name " + name + " from the ban list.");
```

Comment:
Silent not quiet

Reply:
its -q, q for quiet, for some reason it is always referenced in code as silent and not quiet but yeah